js/components/DeploymentComponent.jsx

```diff
@@ -81,27 +99,27 @@ var DeploymentComponent = React.createClass({
       // Set `title` on cells that potentially overflow so hovering on the
       // cells will reveal their full contents.
       <tr>
-        <td className="overflow-ellipsis" title={model.get("id")}>
-          {model.get("id")}
+        <td className="overflow-ellipsis" title={model.id}>
+          {model.id}
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            })}
+            }).value()}
           </ul>
         </td>
         <td>
           <ul className="list-unstyled">
-            {model.get("currentActions").map(function (action) {
+            {lazy(model.currentActions).map(function (action) {
               return <li key={action.app}>{action.action}</li>;
-            })}
+            }).value()}
```

Comment:
What is the purpose of value here? That is afaik only used in combination with chain